Sources/MongoSwift/MongoConnectionString.swift

```diff
@@ -154,6 +168,27 @@ public struct MongoConnectionString: Codable, LosslessStringConvertible {
                     self.authMechanism = try MongoCredential.Mechanism(value)
                 case "authmechanismproperties":
                     self.authMechanismProperties = try self.parseAuthMechanismProperties(properties: value)
+                case "ssl":
```

Comment:
I kind of think we should accept both of these options but only have a `tls` Swift property for it. This would mean that round tripping a non-canonical connection string through `MongoConnectionString` wouldn't work, but IMO that's okay. WDYT? cc @kmahar

Reply:
yeah I think that makes sense for the public API on `MongoConnectionString` (on this `Options` type is fine to keep both since it's internal and used for validation). and it seems acceptable to me as well to do that normalization and only include `tls` if someone round trips a string that specified both `tls` and `ssl`.